cmd/soroban-rpc/internal/ledgerentry_storage/ledgerentry_storage.go

```diff
@@ -0,0 +1,238 @@
+package ledgerentry_storage
```

Comment:
how do you feel about naming the packages `data/ledger` and `data/events` ?

Reply:
I think that the extra level of indirection is not necessary and `ledgerentry_storage` is much more descriptive than `data/ledger`